api/apps/api/src/migrations/api/1691980238543-AddCostSurdaceApiDb.ts

```diff
@@ -8,6 +8,8 @@ export class AddCostSurdaceApiDb1691980238543 implements MigrationInterface {
       project_id uuid NOT NULL references projects(id) ON DELETE CASCADE,
       scenario_id uuid references scenarios(id),
       name VARCHAR NOT NULL,
+      min float8,
+      max float8,
```

Comment:
depending on when we have this data in the overall flow, we may want to make these `NOT NULL` - though if we need to first create a cost surface and then only _at some later point_ we have min,max data then of course, ok as is